plugins/orchestrator-backend/README.md

```diff
@@ -3,3 +3,42 @@
 Welcome to the backend package for the Orchestrator plugin!
 
 For more information about the Orchestrator plugin, see the [Orchestrator Plugin documentation](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/orchestrator) on GitHub.
+
+## Backend to get workflows from Data Index
```

Comment:
Sorry, I just noticed that we changed the `README` file in the `orchestrator-backend` package (in your original PR).
We are actually centralizing the instructions in `plugins/orchestrator/README.md` (including backend).
Could you please move this content to somewhere in
`plugins/orchestrator/README.md#setting-up-the-orchestrator-backend-package` section?

This new change will be merged back to `KOGITO-9778-setup` branch once I do a sync.

Reply:
Now that you've updated `plugins/orchestrator/README.md`, can you please revert the changes in this file? (`plugins/orchestrator-backend/README.md`)